### PR TITLE
Add call to users.conversations

### PR DESCRIPTION
**Pull request by asvoboda**
Implements https://api.slack.com/methods/users.conversations

### CODE REVIEW

**Comment by asvoboda** [comment]
Is there anything else that you'd like me to add to this PR?

File: conversation.go [hunk 1]
```diff
@@ -100,6 +107,41 @@ func (api *Client) GetUsersInConversationContext(ctx context.Context, params *Ge
 	return response.Members, response.ResponseMetaData.NextCursor, nil
 }
 
+// GetConversationsForUser returns the list conversations for a given user
+func (api *Client) GetConversationsForUser(params *GetConversationsForUserParameters) (channels []Channel, nextCursor string, err error) {
+	return api.GetConversationsForUserContext(context.Background(), params)
+}
+
+// GetConversationsForUserContext returns the list conversations for a given user with a custom context
+func (api *Client) GetConversationsForUserContext(ctx context.Context, params *GetConversationsForUserParameters) (channels []Channel, nextCursor string, err error) {
```
Comment (james-lawrence) [inline conversation.go:116]: it'd be better if you returned the entire GetConversationsForUserParameters data for use. slightly easier to use as an API.

**Review by james-lawrence — COMMENTED** [review]
overall looks fine, 1 minor comment. sorry about the delay in reviewing.

File: conversation.go (hunk 1 above)
Comment (asvoboda) [reply conversation.go:116]: It seems like the rest of the conversation.go method have the same api that lets delegates that work to the caller. I didn't want to introduce a new method that functioned differently. I can definitely add another higher level method for the caller to return all data if you'd like.

Comment (james-lawrence) [reply conversation.go:116]: fair point, can always change it later.

File: conversation.go [hunk 2]
```diff
@@ -100,6 +107,41 @@ func (api *Client) GetUsersInConversationContext(ctx context.Context, params *Ge
...
+	}
+	if params.Types != nil {
+		values.Add("types", strings.Join(params.Types, ","))
+	}
+	response := struct {
+		Channels         []Channel        `json:"channels"`
+		ResponseMetaData responseMetaData `json:"response_metadata"`
+		SlackResponse
+	}{}
+	err = post(ctx, api.httpclient, "users.conversations", values, &response, api.debug)
```
Comment (richmondwang) [inline conversation.go:135]: this should be `postSlackMethod`? 
builds are failing now.

Comment (suzuki-shunsuke) [reply conversation.go:135]: I send PR to fix it.

https://github.com/nlopes/slack/pull/401

**Pull request merged**